### PR TITLE
1935 checkbox tests

### PR DESCRIPTION
**Pull request by Sudalius**

### CODE REVIEW

**Comment by mf-code** [comment]
[5512] - aPi8U4T2

**Comment by mf-code** [comment]
[5512] - pfn4xK9o

**Comment by mf-code** [comment]
[5512] - bz6nmJrB

**Comment by mf-code** [comment]
[5512] - c25uHKZt

**Comment by mf-code** [comment]
[5512] - a6jkpIx8

**Comment by mf-code** [comment]
[5512] - giDlyS8M

**Comment by mf-code** [comment]
[5512] - D2H77NTQ

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/CheckboxTests.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        configurableCheckedCheckbox.check();
+        configurableResultCheckbox.isSelected();
+        configurableCheckedCheckbox.uncheck();
+        configurableResultCheckbox.is().deselected();
+
+        configurableIndeterminateCheckbox.check();
+        configurableResultCheckbox.hasClass("mat-checkbox-indeterminate");
+
+        configurableCheckboxAlignBeforeRadioButton.click();
+        configurableResultCheckbox.hasClass("mat-checkbox-label-before");
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/CheckboxTests.java:40]: Please add a check for configurableCheckboxAlignAfterRadioButton

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/CheckboxTests.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    @Test
+    public void configurableCheckboxTest() {
+        configurableCheckedCheckbox.show();
+        configurableCheckedCheckbox.check();
+        configurableResultCheckbox.isSelected();
+        configurableCheckedCheckbox.uncheck();
+        configurableResultCheckbox.is().deselected();
+
+        configurableIndeterminateCheckbox.check();
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/CheckboxTests.java:36]: A suuggestion: consider splitting this test into three smaller tests: on Checked\Indeterminate checkboxes, on alignment radios, on disabled checkbox?

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/CheckboxTests.java (hunk 1 above)
Comment (Sudalius) [reply jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/CheckboxTests.java:40]: Done

**Comment by mf-code** [comment]
[5553] - ey9mrQR1

**Comment by mf-code** [comment]
[5553] - pcOIQ3bK

**Comment by mf-code** [comment]
[5553] - e9OtgJAl

**Comment by mf-code** [comment]
[5553] - ZxMQpF5f

**Comment by mf-code** [comment]
[5553] - Allure report on Netlify: https://5ed92203940136f7796d8476--musing-swartz-6fc060.netlify.app

**Pull request merged**